j2/vars/custom-sap-cap-rules.yaml

```diff
@@ -313,13 +290,86 @@ rules:
       - label: Object.Data_Source
         props:
           Name: "SAP/C@P CUSTOMER PROSPECT"
-#            excludeFromSearch: true
-#          Type:
-#            val: "sap/c@p Customer Prospect"
-#            excludeFromSearch: true
-#          Description:
-#            val: "Registros da tabela Customer Prospect no SAP/C@P"
-#            excludeFromSearch: true
+
+      - label: Event.Consent
+        condition: "${Mail_Consent?.trim()?.length() > 0}"
+        name: mail_consent
+        props:
+          Created_By: "${Created_By?.trim()}"
+          Changed_By: "${Changed_by?.trim()}"
+          Metadata.Create_Date:
+            val: "${Created_on?.replaceAll('\\\\.','/')?.trim()?:'01/01/1666'}"
+            type: "java.util.Date"
+          Metadata.Update_Date:
+            val: "${Changed_on?.replaceAll('\\\\.','/')?.trim()?:'01/01/1666'}"
+            type: "java.util.Date"
+          Description: "Mail Consent"
+          Status: "${Mail_Consent?.trim()?.equals('Y')?'Consent':'No Consent'}"
+          Data_Origin: "${Data_Origin?.trim()}"
+
+      - label: Event.Consent
+        condition: "${E_Mail_Consent?.trim()?.length() > 0}"
+        name: email_consent
+        props:
+          Created_By: "${Created_By?.trim()}"
+          Changed_By: "${Changed_by?.trim()}"
+          Metadata.Create_Date:
+            val: "${Created_on?.replaceAll('\\\\.','/')?.trim()?:'01/01/1666'}"
+            type: "java.util.Date"
+          Metadata.Update_Date:
+            val: "${Changed_on?.replaceAll('\\\\.','/')?.trim()?:'01/01/1666'}"
+            type: "java.util.Date"
+          Description: "E-Mail Consent"
+          Status: "${E_Mail_Consent?.trim()?.equals('Y')?'Consent':'No Consent'}"
+          Data_Origin: "${Data_Origin?.trim()}"
+
+      - label: Event.Consent
+        condition: "${Telephone_Consent?.trim()?.length() > 0}"
+        name: telephone_consent
+        props:
+          Created_By: "${Created_By?.trim()}"
+          Changed_By: "${Changed_by?.trim()}"
+          Metadata.Create_Date:
+            val: "${Created_on?.replaceAll('\\\\.','/')?.trim()?:'01/01/1666'}"
+            type: "java.util.Date"
+          Metadata.Update_Date:
+            val: "${Changed_on?.replaceAll('\\\\.','/')?.trim()?:'01/01/1666'}"
+            type: "java.util.Date"
+          Description: "Telephone Consent"
+          Status: "${Telephone_Consent?.trim()?.equals('Y')?'Consent':'No Consent'}"
+          Data_Origin: "${Data_Origin?.trim()}"

```

Comment:
Create a new property with Customer_Id to ensure that the event vertices are unique